### PR TITLE
Add Swift examples to social docs

### PR DESCRIPTION
**Pull request by Bucimis**

### CODE REVIEW

File: _docs/_developer_guide/platform_integration_guides/ios/analytics/social_data_tracking.md [hunk 1]
```diff
@@ -30,8 +30,8 @@ ABKFacebookUser *facebookUser = [[ABKFacebookUser alloc] initWithFacebookUserDic
 {% tab swift %}
```
Comment (hokstuff) [inline _docs/_developer_guide/platform_integration_guides/ios/analytics/social_data_tracking.md]: Line 26 above has 2 extra spaces at the beginning of the code snippet
`[Appboy sharedInstance].user.facebookUser = facebookUser;`

File: _docs/_developer_guide/platform_integration_guides/ios/analytics/social_data_tracking.md [hunk 2]
```diff
@@ -49,10 +49,24 @@ Appboy.sharedInstance()?.user.facebookUser = facebookUser;
...
 ```objc
 ABKFacebookUser *facebookUser = [[ABKFacebookUser alloc] initWithFacebookUserDictionary:facebookUserPublicProfile numberOfFriends:-1 likes:nil];  
 ```
 
+{% endtab %}
+{% tab swift %}
+
+```swift
+let facebookUser = ABKFacebookUser(facebookUserDictionary: facebookUserDictionary, numberOfFriends: -1, likes:nil)
+Appboy.sharedInstance()?.user.facebookUser = facebookUser
```
Comment (hokstuff) [inline _docs/_developer_guide/platform_integration_guides/ios/analytics/social_data_tracking.md]: To keep it consistent with the ObjC snippet, we can either remove line 64 or add the same thing to the ObjC code

File: _docs/_developer_guide/platform_integration_guides/ios/analytics/social_data_tracking.md (hunk 1 above)
Comment (Bucimis) [reply _docs/_developer_guide/platform_integration_guides/ios/analytics/social_data_tracking.md]: fixed

File: _docs/_developer_guide/platform_integration_guides/ios/analytics/social_data_tracking.md (hunk 2 above)
Comment (Bucimis) [reply _docs/_developer_guide/platform_integration_guides/ios/analytics/social_data_tracking.md]: fixed, good call

**Pull request merged**